docker-compose.yml

```diff
@@ -16,9 +16,11 @@ services:
         condition: service_started
     expose:
       - 8080
+    env_file:
+      - .env
     environment:
       FIDES__CONFIG_PATH: "/fides/.fides/fides.toml"
-      FIDES__CLI__ANALYTICS_ID: ${FIDES__CLI__ANALYTICS_ID}
+      FIDES__CLI__ANALYTICS_ID: ${FIDES__CLI__ANALYTICS_ID:-}
```

Comment:
Bonus: this suppresses those "WARNING" logs in docker compose, since we explicitly treat these as optional vars.